wiki/Linux-Troubleshooting.md

```diff
@@ -11,34 +11,47 @@ This path might differ based on your Steam installation, in that case SteamVR wi
 Then pick the storage location with the star emoji (⭐) and take the path directly above the usage statistics. Prepend this path to `steamapps/common/SteamVR/bin/vrmonitor.sh`.
 Finally put this entire path into the SteamVR commandline options instead of the other one.
 
-### Hyprland/Sway/Wlroots Fix
+### Hyprland/Sway/Wlroots Qt fix
 
```

Comment:
Does this not do the exact same thing as the other xwayland fix just through a different mechanism or am I missing something (ik you didn't change it I just never got around to address it)

Reply:
Afaik its a bit different thing and specific to wlroots, not sure what exactly but other De's are not really affected

Reply:
then it's probably some mess due to steamvr being built on 50 different things at the same time